designsafe/apps/applications/views.py

```diff
@@ -274,7 +274,7 @@ def call_api(request, service):
                                     'query': query
                                 }
                             })
-                            data = agave.meta.listMetadata(q=query)
+                            data = agave.meta.listMetadata(q=query, limit=0)
```

Comment:
Why `limit=0`?

Reply:
`limit=0` allows for unlimited responses from the query - before in the applications view we were only getting the default max

Reply:
Probably better to write a method to paginate this.
Although the metadata records don't grow drastically from one day to another and we still need to re-write this entire view class.
Let me know if you have time to write a paginator for this. If not, please create a task for it so we can do it in the future, eventually.

Reply:
I agree 100%. I'll see if I can get around to it with these class changes, but will probably have to make a task for the future - need to get to transition to core

Reply:
👍 Sounds good.